controllers/v1/users.js

```diff
@@ -84,7 +84,90 @@ module.exports = class Users {
 				return resolve(targetedSolutions)
 			} catch (error) {
 				return reject({
-					status: error.status || HTTP_STATUS_CODE['internal_server_error'].status,
+					status: error.status || HTTP_STATUS_CODE.internal_server_error.status,
+
+					message: error.message || HTTP_STATUS_CODE.internal_server_error.message,
+
+					errorObject: error,
+				})
+			}
+		})
+	}
+
+	/**
+      * @api {post} /project/v1/users/programs?isAPrivateProgram=true&page=:page&limit=:limit&search=:search 
```

Comment:
@Prajwal17Tunerlabs Please add validation for this api

Reply:
validation is added